components/common/labels/Grid.tsx

```diff
@@ -28,19 +22,10 @@ export const LabelSelector = (props: LabelsProps) => {
     })),
   )
   const allOptions = buildAllLabelOptions(defaultLabels, options)
-  const groupedLabelList = groupLabelList(allOptions)
-  const selectorOptions = Object.entries(groupedLabelList).map(
-    ([group, groupInfo]) => ({
-      label: group,
-      options: groupInfo.labels.map((label) => {
-        const labelText = typeof label === 'string' ? label : label.id
-        return {
-          label: labelText,
-          value: labelText,
-        }
-      }),
-    }),
-  )
+  const selectorOptions = Object.values(ALL_LABELS).map((labelOption) => ({
+    label: labelOption.identifier,
```

Comment:
Any idea what this new type error is?

Reply:
(I assume this is related to @atproto/api updates.)

Reply:
yep, cleaning it up now. I was referencing a local version of the api package so had to adjust things to get the app to run.